rules/private/phases/phase_classpaths.bzl

```diff
@@ -58,7 +58,12 @@ def phase_classpaths(ctx, g):
     ).transitive_runtime_jars
 
     srcs = [file for file in ctx.files.srcs if file.extension.lower() in ["java", "scala"]]
-    src_jars = [file for file in ctx.files.srcs if file.extension.lower() in ["srcjar"]]
+    src_jars = [
+        file
+        for file in ctx.files.srcs
+        if file.path.endswith("srcjar") or file.path.endswith("-sources.jar") or
```

Comment:
is the `.lower()` not necessary for the `file.path` like it is for the `file.extension`?

Reply:
Good catch. I'll fix and see if I can add a test for it.

Reply:
I've updated these to be consistent with the others and have `lower()`

That said, looks like allow_files is not case insensitive, so I can't add a test. Which makes me question whether we want the lower here, but I suppose it won't hurt.